src/tooling/docs-builder/Commands/ReleaseNotesCommand.cs

```diff
@@ -0,0 +1,96 @@
+// Licensed to Elasticsearch B.V under one or more agreements.
+// Elasticsearch B.V licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information
+
+using ConsoleAppFramework;
+using Documentation.Builder.Arguments;
+using Elastic.Documentation.Configuration;
+using Elastic.Documentation.Diagnostics;
+using Elastic.Documentation.Services;
+using Elastic.Documentation.Services.ReleaseNotes;
+using Microsoft.Extensions.Logging;
+
+namespace Documentation.Builder.Commands;
+
+internal sealed class ReleaseNotesCommand(
+	ILoggerFactory logFactory,
+	IDiagnosticsCollector collector,
+	IConfigurationContext configurationContext
+)
+{
+	/// <summary>
+	/// Release notes commands. Use 'release-notes create' to create a new changelog fragment.
+	/// </summary>
+	[Command("")]
+	public Task<int> Default()
+	{
+		collector.EmitError(string.Empty, "Please specify a subcommand. Use 'release-notes create' to create a new changelog fragment. Run 'release-notes create --help' for usage information.");
+		return Task.FromResult(1);
+	}
+
+	/// <summary>
+	/// Create a new release notes changelog fragment from command-line input
+	/// </summary>
+	/// <param name="title">Required: A short, user-facing title (max 80 characters)</param>
+	/// <param name="type">Required: Type of change (feature, enhancement, bug-fix, breaking-change, etc.)</param>
+	/// <param name="products">Required: Products affected in format "product target lifecycle, ..." (e.g., "elasticsearch 9.2.0 ga, cloud-serverless 2025-08-05")</param>
```

Comment:
I wonder `applies_to` could be the base for it.
Not sure if we can align the syntax. (Also applies_to must be extended to also support dates)

I guess we cannot do this now.. but aligning this in the future could be nice. 

I definitely see a connection.

Reply:
> ++ I don't think we need to put dates any longer. Since in our proposal we would discover what serverless release these go out.

I've left that item in the schema for now since at the moment we don't have the longer-term solution.